### PR TITLE
Retrieve Country Currency based on Location

### PR DESCRIPTION
**Pull request by chandachewe10**
Retrieving Country Currency Based on users Location for the Default Driver (IpAPI). This is useful when for example you are creating a billing Invoice Portal where different users in different countries will be required to generate an Invoice on your system. Upon retrieving the country location the country currency will be imported on the Invoice automatically for that user:
 
`$position->currencyCode`

### CODE REVIEW

File: src/Drivers/IpApi.php [hunk 1]
```diff
@@ -13,7 +13,7 @@ class IpApi extends Driver
      */
     protected function url($ip)
     {
-        return "http://ip-api.com/json/$ip";
+        return "http://ip-api.com/json/$ip?fields=status,message,country,countryCode,region,regionName,city,zip,lat,lon,timezone,currency,isp,org,as,query";
```
Comment (stevebauman) [inline src/Drivers/IpApi.php:16]: The `fields` parameter doesn't appear to be necessary to return the `currencyCode`.

**Review by stevebauman — CHANGES_REQUESTED** [review]

File: src/Drivers/IpApi.php (hunk 1 above)
Comment (chandachewe10) [reply src/Drivers/IpApi.php:16]: So the `IpApi Driver` by default does not return the `currency code` unless specified in the fields status. Hence to return the `currence code` I have to specify in the fields status together with other default response.

**Review by chandachewe10 — COMMENTED** [review]
Added currency retrieval for other two drivers `IpData` and 'Kloudend`. Removed unnecessary whitespaces.

File: src/Position.php [hunk 2]
```diff
@@ -20,6 +20,13 @@ class Position implements Arrayable
      */
     public $countryName;
 
+    /**
+     * The country Currency.
```
Comment (stevebauman) [inline src/Position.php:24]:
```suggestion
     * The country's currency code.
```

File: src/Drivers/IpApi.php (hunk 1 above)
Comment (stevebauman) [reply src/Drivers/IpApi.php:16]: Ah gotcha, you're right @chandachewe10, my mistake! Thanks for clarifying this 🙏

**Review by stevebauman — APPROVED** [review]

**Comment by stevebauman** [comment]
Tests are failing @chandachewe10. Would you be able to update them?

**Comment by chandachewe10** [comment]
> Tests are failing @chandachewe10. Would you be able to update them?

@stevebauman I think the Tests are good to run now

File: src/Position.php [hunk 3]
```diff
@@ -22,6 +22,13 @@ class Position implements Arrayable
      */
     public ?string $countryName = null;
 
+    /**
+     * The country's currency code.
+     *
+     * @var string|null
+     */
+    public $currencyCode;
```
Comment (stevebauman) [inline src/Position.php:30]:
```suggestion
    /**
     * The country's currency code.
     */
    public ?string $currencyCode;
```

**Pull request merged**